### DISCUSSION TITLE
Can testcontainers be used in Github Actions?

### DISCUSSION THREAD

**Discussion by shwarcu** [testcontainers/testcontainers-node#500] [Q&A] [answered]
Hi, 
I am considering using `testcontainers-node` for automated tests of our service that uses postgres and redis. I am wondering if it's supported to run tests using `testcontainers-node` in github actions environment.
Is this generally allowed practice? Has anyone tested it?

**Comment by cristianrgreco** [comment, marked as answer]
Hi @shwarcu, yes! The build pipeline for testcontainers-node runs in GitHub actions 🙂
